### DISCUSSION TITLE
[아파치 카프카] - 들어가기에 앞서

### DISCUSSION THREAD

**Discussion by Irisation23** [cs-collections/cs-collections-discussion#16] [k. 대규모 서비스에 필요한 기술]
# 서문

- 카프카는 이벤트 스트리밍뿐만 아니라 빅데이터 아키텍처에 있어서 중요한 역할을 하는 만큼 `대용량`, `대규모 데이터 처리`를 위한 플랫폼이다.
- 카프카만의 개념과 용어를 따로 정리해야 할 필요가 있다.

정리하게 될 내용은 아래와 같다.
아래의 내용들은 향후 추가될 때 마다 링크에 추가 할 예정이다.

- 카프카를 왜 사용해야 하는가?
- 카프카의 브로커는 어떤 역할을 하는가?
- 컨슈머와 프로듀서란 무엇인가?
- 카프카 커넥트와 커넥터란 무엇인가?
- 스프링 카프카란 무엇인가?
- 미러메이커2란 무엇인가?
- 클라우드 기반 카프카는 무엇이 있고, 어떻게 사용하는가?

**Comment by JoisFe** [comment]
아무런 개념도 없이 무작정 rabbit Mq를 같이 사용하였는데 이번 기회에 카프카를 공부하시는 것 같네요 ㅎㅎ
열심히 공부해서 많이 알려주십쇼^^
